README.rst

```diff
@@ -219,7 +219,7 @@ installed and in your PATH.
 
 LICENSE
 -------
-Python-tesseract is released under the GPL v3.
+Check the LICENSE file included in the Python-tesseract repository/distribution.
```

Comment:
Can we update this to say that as of this version 0.3.1 it is Apache?

Reply:
Yep

Reply:
Done, republished the 0.3.1 release including the notice in the LICENSE section.